docs/100.yaml

```diff
@@ -55,6 +55,8 @@
   author : Yury G. Kudryashov (first) and Benjamin Davidson (second)
 16:
   title  : Insolvability of General Higher Degree Equations
+  decl  : exists_not_solvable_by_rad
```

Comment:
For links to the archive you need to give a long form URL. We might want to change this.
Anyway, see some of the other entries for the format.